srcpkgs/teams/files/teams-wrapper.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+LD_LIBRARY_PATH=/usr/lib/teams exec /usr/lib/teams/teams "$@"
```

Comment:
Can `patchelf` not be used to set the `rpath` of the teams executable, rather than wrapping?